### PR TITLE
making minor change to WebVR GroupData

### PR DESCRIPTION
**Pull request by chrisdavidmills**

### CODE REVIEW

File: macros/GroupData.json [hunk 1]
```diff
@@ -1969,6 +1969,7 @@
                             "VRPose",
                             "VREyeParameters",
                             "VRFieldOfView",
+                            "VRLayer"
```
Comment (Elchi3) [inline macros/GroupData.json]: This is a dictionary. I know we haven't really made our minds up with interfaces vs. dictionaries, but I'm rather hesitant about adding it to a list of "interfaces". What do you think? Is it an important dictionary?

Comment (chrisdavidmills) [inline macros/GroupData.json]: In the newest spec, it is not defined as a dictionary, and even has its own constructor: https://w3c.github.io/webvr/#interface-vrlayer

Comment (Elchi3) [inline macros/GroupData.json]: ha, awesome!

Comment (Elchi3) [inline macros/GroupData.json]: But this line needs a comma

Comment (chrisdavidmills) [inline macros/GroupData.json]: Gah, I'm such a useless asshole sometimes ... comma added ;-)

**Pull request merged**